apps/test/unit/applab/FirebaseStorageTest.js

```diff
@@ -0,0 +1,91 @@
+import { expect } from '../../util/configuredChai';
+import FirebaseStorage from '../../../src/applab/firebaseStorage';
+import { getDatabase, getFirebase } from '../../../src/applab/firebaseUtils';
+
+describe('FirebaseStorage', () => {
+  beforeEach(() => {
+    window.Applab = {
+      channelId: "test-firebase-channel-id",
+      firebaseName: 'test-firebase-name',
+      firebaseAuthToken: 'test-firebase-auth-token',
+    };
+  });
+
+  describe('populateTable', () => {
+    const EXISTING_TABLE_DATA = {
+      cities: {
+        records: {
+          1: '{"city":"New York","state":"NY","id":1}'
+        }
+      }
+    };
+    const NEW_TABLE_DATA_JSON =
+      '{' +
+      '  "cities": [' +
+      '    {"city": "Seattle", "state": "WA"},' +
+      '    {"city": "Chicago", "state": "IL"}' +
+      '  ]' +
+      '}';
+    const NEW_TABLE_DATA = {
+      cities: {
+        records: {
+          1: '{"city":"Seattle","state":"WA","id":1}',
+          2: '{"city":"Chicago","state":"IL","id":2}',
+        }
+      }
+    };
+    function verifyTable(expectedTablesData) {
+      return getDatabase(Applab.channelId).child(`storage/tables`).once('value')
+        .then(snapshot => {
+          expect(snapshot.val()).to.deep.equal(expectedTablesData);
+        }, error => {
+          console.warn(`verifyTable promise rejected: ${error}`);
```

Comment:
You use `console.warn` in error handlers several times in these tests.  Should the tests fail if these ever occur?  If so, throw instead?